src/DataFrames.jl

```diff
@@ -128,4 +128,18 @@ for (dir, filename) in [
     include(joinpath(dir, filename))
 end
 
+# Import  "StringEncodings" if available and set up decoder
+_decoder=nothing
+ENCODING_PACKAGE_READY = false
+function __init__()
+    try Pkg.installed("StringEncodings")
```

Comment:
If DataFrames were to get precompiled including optional functionality but then that optional package were later removed, the behavior prior to 0.5.1 would be unable to load the precompiled copy. At least that bug should be fixed now on 0.5.2 and 0.6.